app/src/main/AndroidManifest.xml

```diff
@@ -46,6 +42,10 @@
         </activity>
 
         <receiver android:name=".etc.ShareBroadcastReceiver" />
+
+        <activity
```

Comment:
It might be just me, but I think it's better to "group" activity with another activities tag (rather than separated with `receiver` in this case).